perception/detection_by_tracker/include/detection_by_tracker/utils.hpp

```diff
@@ -39,6 +39,8 @@ double get2dPrecision(
 double get2dRecall(
   const autoware_auto_perception_msgs::msg::DetectedObject & source_object,
   const autoware_auto_perception_msgs::msg::DetectedObject & target_object);
+geometry_msgs::msg::Polygon rotatePolygon(
+  const geometry_msgs::msg::Polygon & polygon, const double & angle);
```

Comment:
```suggestion
  const geometry_msgs::msg::Polygon & polygon, const double angle);
```

Reply:
@wep21 
I think it is same calculation cost, but is there any reason why we shouldn't use pass-by-reference when it is primitives?

Reply:
It may be my preference, but I feel it is useless to use reference for primitives.

Reply:
Please refer to https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#f16-for-in-parameters-pass-cheaply-copied-types-by-value-and-others-by-reference-to-const.

Reply:
@wep21 @kenji-miyake Thank you :+1: 
BTW, it was same calculation cost in my test environment